LogMonitor/src/LogMonitor/Utility.cpp

```diff
@@ -306,3 +306,22 @@ void Utility::SanitizeJson(_Inout_ std::wstring& str)
         i++;
     }
 }
+
+/// <summary>
+/// Comparing wstrings with ignoring the case
+/// </summary>
+/// <param name="stringA"></param>
+/// <param name="stringB"></param>
+/// <returns></returns>
+bool Utility::CompareWStrings(wstring stringA, wstring stringB)
+{
+    return stringA.size() == stringB.size() && 
+        equal(
+            stringA.cbegin(), 
+            stringA.cend(), 
+            stringB.cbegin(),
+            [](wstring::value_type l1, wstring::value_type r1) { 
+                return towupper(l1) == towupper(r1); 
+            }
+    );

```

Comment:
nit: FAIW, can align the closing `);` with the `equal` on L319.

Reply:
This has been addressed, Had to use https://codebeautify.org/cpp-formatter-beautifier to get the correct formatting

Reply:
can't see the change from my end though, since aligned with `return` instead of `equal`.

Reply:
This should be fine:
![image](https://user-images.githubusercontent.com/28774968/221144820-6860ff0b-c793-4637-9ecf-50f07f4a9023.png)

if it was aligned to return it would look like this:
![image](https://user-images.githubusercontent.com/28774968/221144982-03f6415b-269b-40ee-bf98-4d782a40512a.png)


Try to change the tabbing and see. 

also if you copy the code to an online formatter resource, it should check out.

Reply:
Tabbing issue addressed